exercises/codec-server/practice/codec-server/main.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"strconv"
+
+	// TODO Part A: Import the `edu_converters_go_code` package used by the other code samples.
```

Comment:
I agree — the issue is that VSCode's default Go linting is very strict, and will delete unused module imports upon saving otherwise. It doesn't do this for any other language out of the box, but I suspect 80%+ of our learners will be using VSCode, and even if I override this in my own environment, they could otherwise wind up with an unused import getting deleted the first time they try editing the file in any other way and saving it, and then not noticing it's missing.